cypress-io/cypress
# chore: use `experimentalSingleTabRunMode` in all of our monorepo packages and system tests that are configured for CT

- Closes #25165

### User facing changelog


n/a

### Additional details


We are [moving `experimentalSingleTabRunMode` into GA](https://github.com/cypress-io/cypress/issues/25166) at some point. Before we do this, we should use it ourselves to test it out as the default. This should make it easier to come across any edge cases, perf issues, etc. before we make it the default for everyone.

### Steps to test


Not much to test, just take a look at the CI run and make sure that everything is passing. Note that I didn't make any changes to the `pristine` projects in `system-tests/` since these are used to test our scaffolding logic.

### How has the user experience changed?


n/a

### PR Tasks


- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
